doc/source/tutorials/performance-factors.rst

```diff
@@ -0,0 +1,87 @@
+
+Space tiles
+    In dense arrays, space tile shape and size determines the atomic unit of I/O and compression. All space tiles that intersect a read query's subarray are read from disk whole, even if they only partially intersect with the query. Matching space tile shape and size closely with the typical read query can greatly improve performance.
+
+    In sparse arrays, space tiles determine the minimum bounding rectangle (MBR) shape and size. All tiles in a sparse array whose MBRs intersect a read query's subarray are read from disk. Reducing the space tile shape and size in sparse arrays can reduce the number of overlaps across MBRs in the array, which can improve read performance as fewer MBRs may then overlap with the typical read query.
+
+Tile capacity
+    In sparse arrays, the tile capacity determines the number of cells in a data tile, which is the atomic unit of I/O and compression. If the capacity is too small, the data tiles can be too small, requiring many small, independent read operations to fulfill a particular read query. If the capacity is too large, I/O operations would become fewer and larger (a good thing in general), but the amount of wasted work may increase due to reading more data than necessary to fulfill the typical read query. Tile capacity does not apply to dense arrays.
+
+Dimensions vs. attributes
+    If read queries to the array typically access the entire domain of a particular dimension, consider making that dimension an attribute instead. Dimensions should be used when read queries will issue ranges of that dimension of cells to retrieve, which facilitates TileDB internally pruning cell ranges or whole tiles from the data that must be read. Additionally, it is not possible to subselect on dimensions (i.e. a read query can specify only a subset of attributes should be read, but it must always specify all dimensions).
+
+Compression
+    Compression of attributes can reduce persistent storage consumption, but necessarily increases the time required to read or write tiles in the array. However, tiles on disk that are smaller due to compression can be read from disk in less time, which can improve performance. Finally, many compressors offer different levels of compression, which can be used to fine-tune the tradeoff.
+
+Dense vs. sparse arrays
+    Choosing a dense or sparse array representation depends on the particular data and use case. Sparse arrays in TileDB have additional overhead versus dense arrays due to the need to explicitly store cell coordinates of the non-empty cells. For a particular use case, this extra overhead may be outweighed by the storage savings of not storing dummy values for empty cells in the dense alternative, in which case a sparse array would be a good choice.
```

Comment:
I would push Dense vs Spare to be the first perf. decision :)

Another difference is that sparse cells (potentially) need to be sorted, which has extra performance cost.

Reply:
I guess you address this in the read / write query section.